client/js/middleware/serialization/serializer_utils.js

```diff
@@ -0,0 +1,8 @@
+import _    from 'lodash';
+
+export function scrub(item) {
+  let scrubbedItem = _.cloneDeep(item);
+  scrubbedItem = _.omitBy(scrubbedItem, _.isNil);
+  scrubbedItem = _.omitBy(scrubbedItem, _.isEmpty);
```

Comment:
Are you sure, is empty returns false on numbers. it would rip empty strings out though